docs/reference/cli/index.md

```diff
@@ -2178,10 +2178,50 @@ p2p-static-peers: ["/ip4/151.150.191.80/tcp/9000/p2p/16Ui...aXRz",
                     "/ip4/151.150.191.80/tcp/9000/p2p/16Ui...q6f1"]
 ```
 
+List of comma-separated [multiaddresses](https://docs.libp2p.io/concepts/appendix/glossary/#multiaddr) of static peers
+with which to establish and maintain connections.
+
+  </TabItem>
+</Tabs>
+
+### `p2p-direct-peers`
```

Comment:
We keep an alphabetical order in the docs, so this should go above `p2p-discovery-bootnodes`